samples/ResourceOwnerPasswordFlow/Certificate.cer

```diff
@@ -0,0 +1,24 @@
+Bag Attributes
```

Comment:
Core CLR is clearly the future, as it will be the only target supported by Windows Server 2016 Nano and will likely replace Mono on Linux/OS X in the next few months. App containers like Docker are more and more popular and attracts many .NET developers to Linux, where Mono is still the most complete option, so yeah, these targets are definitely important.

The question is: are you ready to include ugly hacks in your PR (and potentially in your tutorial) to support them? :smile: 

/cc @AlexZeitler